Wikipedia/Localizations/skr-arab.lproj/Localizable.strings

```diff
@@ -0,0 +1,146 @@
+"dark-theme-display-name" = "گہرا";
+"dim-images" = "مدھم تصویر";
+"edit-summary-choice-other" = "ٻئے";
+"empty-no-search-results-message" = "کوئی نتیجہ نی لبھا";
+"explore-continue-reading-heading" = "پڑھݨ جاری رکھو";
+"explore-nearby-placeholder-cancel" = "منسوخ";
+"explore-nearby-placeholder-dismiss" = "مکاؤ";
+"explore-nearby-placeholder-heading" = "جاہیں";
+"explore-nearby-placeholder-sub-heading-on-wikipedia" = "وکیپیڈیا تے";
+"field-alert-token-invalid" = "غلط کوڈ";
+"field-alert-username-unavailable" = "ورتݨ آلا ناں موجود کائنی";
+"field-captcha-placeholder" = "کیپچا عبارت";
+"field-email-title" = "ای میل";
+"field-email-title-optional" = "ای میل (مرضی ہے)";
+"field-new-password-confirm-placeholder" = "نواں پاس ورڈ ولدا لکھو";
+"field-new-password-confirm-title" = "نویں پاس ورڈ دی تصدیق کرو";
+"field-new-password-placeholder" = "نواں پاس ورڈ درج کرو";
+"field-new-password-title" = "نواں پاس ورڈ";
+"field-password-confirm-placeholder" = "پاس ورڈ ولدا لکھو";
+"field-password-confirm-title" = "پاس ورڈ دی تصدیق کرو";
+"field-password-placeholder" = "پاس ورڈ درج کرو";
+"field-password-title" = "پاس ورڈ";
+"field-token-title" = "تصدیقی کوڈ";
+"field-username-placeholder" = "ورتݨ آلا ناں درج کرو";
+"field-username-title" = "ورتݨ آلا ناں";
+"find-in-page-button-label" = "ورقے وچ ڳولو";
+"find-in-page-number-matches" = "$1 / $2";
+"forgot-password-button-title" = "ولدا ٹھیک کرو";
+"forgot-password-title" = "پاس ورڈ ولدا ٹھیک کرو";
+"forgot-password-username-or-email-title" = "یا";
+"gdfl" = "جی ڈی ایف ایل";
+"history-clear-all" = "صاف";
+"history-clear-cancel" = "مکاؤ";
+"history-clear-delete-all" = "جیا سبھ مٹاؤ";
+"history-section-today" = "اڄ";
+"history-section-yesterday" = "کل";
+"history-title" = "تاریخچہ";
+"hockeyapp-alert-always-send" = "ہمیشہ بھیجو";
+"hockeyapp-alert-do-not-send" = "نہ بھیجو";
+"hockeyapp-alert-privacy" = "$1 رازداری";
+"hockeyapp-alert-send-report" = "رپورٹ بھیجو";
+"home-hide-suggestion-cancel" = "منسوخ";
+"home-title" = "پھرولو";
+"icon-shortcut-search-title" = "وکیپیڈیا وچ ڳولو";
+"in-the-news-notification-read-now-action-title" = "ہُݨ پڑھو";
+"in-the-news-notification-share-action-title" = "شیئر";
+"info-box-close-text" = "بند کرو";
+"languages-title" = "زبان وٹاؤ";
+"main-menu-account-login" = "لاگ ان تھیوو";
+"main-menu-account-logout" = "لاگ آؤٹ";
+"main-menu-account-logout-cancel" = "منسوخ";
+"main-menu-nearby" = "نیڑے";
+"main-menu-privacy-policy" = "پرائیویسی پالیسی";
+"main-menu-title" = "ٻئے";
+"menu-cancel-accessibility-label" = "منسوخ";
+"menu-trash-accessibility-label" = "مٹاؤ";
+"navbar-title-mode-edit-wikitext-preview" = "پہلا ݙکھالا";
+"nearby-distance-label-feet" = "$1 فٹ";
+"nearby-distance-label-km" = "$1 کلو میٹر";
+"nearby-distance-label-meters" = "$1 میٹر";
+"nearby-distance-label-miles" = "$1 میل";
+"new-password-title" = "پاس ورڈ درج کرو";
+"number-billions" = "$1 ارب";
+"number-millions" = "$1 ملین";
+"number-thousands" = "$1 ہزار";
+"on-this-day-detail-header-date-range" = " $1 توں - $2";
```

Comment:
@joewalsh 
hmm is this `$1 2$` just gh having rtl issues?

Reply:
@montehurd looks like it. Either way, `skr-arab` isn't supported by iOS.